File: calculo1rm.py
def calcula1rm (carga, reps, rpe):
    #Repetições em reserva
    rir = 10 - rpe
    #Coeficiente do calculo
    coef = 0.03

    #Calculo de RM
    primeiro = (36 / (37 - reps)) 
    segundo = carga * primeiro
    terceiro = rir * coef
    quarto = 1 - terceiro
    rmestimado = (segundo + 6) / quarto

    return round(rmestimado, 2)

#Chama função e coloca os inputs para as variaveis
resultado = calcula1rm(carga = int(input("Digite a carga levantada (em kg): ")),
reps = int(input("Digite o número de repetições realizadas: ")),
rpe = float(input("Digite o RPE (de 4 a 10): ")))

#Mostra resultado em kg
print(f"Parabens seu nome 1 REP MAX é: {resultado} kg")

File: calculo1rm.js
function calcula1RM(carga, reps, rpe) {
    let rir = 10 - rpe;
    let coef = 0.03;
    let primeiro = (36 / (37 - reps));
    let segundo = carga * primeiro;
    let terceiro = rir * coef;
    let quarto = 1 - terceiro;
    let rmestimado = (segundo + 6) / quarto;

    return rmestimado.toFixed(2);
  }

  document.getElementById('rm-form').addEventListener('submit', function(event) {
    event.preventDefault();

    const carga = parseFloat(document.getElementById('carga').value);
    const reps = parseFloat(document.getElementById('reps').value);
    const rpe = parseFloat(document.getElementById('rpe').value);

    if (isNaN(carga) || isNaN(reps) || isNaN(rpe) || carga <= 0 || reps <= 0 || rpe <= 0) {
        document.getElementById('onerepmax').innerText = 'Por favor, insira valores válidos.';
        return;
    }

    const onerepmax = calcula1RM(carga, reps, rpe);
    document.getElementById('onerepmax').innerText = `${onerepmax} kg`;
  });